spec/presenters/statistics_presenter_spec.rb

```diff
@@ -35,17 +35,61 @@
 
     context 'when services are enabled' do
       before do
-        AppConfig.privacy.statistics.user_counts = true
-        AppConfig.privacy.statistics.post_counts = true
-        AppConfig.privacy.statistics.comment_counts = true
         AppConfig.services = {
-          "facebook" => {"enable" => true},
+          "facebook" => {"enable" => true, "authorized" => true},
           "twitter" => {"enable" => true},
           "wordpress" => {"enable" => false},
-          "tumblr" => {"enable" => false}
+          "tumblr" => {"enable" => true, "authorized" => false}
+        }
+      end
+
+      it 'provides services in json' do
+        expect(@presenter.as_json).to eq({
+           "name" => AppConfig.settings.pod_name,
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.